nntrainer/include/layer.h

```diff
@@ -123,7 +123,7 @@ class Layer {
       type(LAYER_UNKNOWN),
       loss(0.0),
       cost(COST_UNKNOWN),
-      activation_type(ACT_UNKNOWN),
+      activation_type(ACT_NONE),
```

Comment:
Please note that, ACT_NONE will set an layer with identity activation function w.r.t. current implementation of `activation layer` in `neuralnet::_make_act_layer`

Reply:
Right, I updated to result in an error if the activation type is unknown.
IMO itis better to make it a no-op (no activation) than keep it unknown. Regarding the overhead, unit layers can be optimized later.
Can update it to `unknown` if you think its better.

Reply:
I think `ACT_NONE` is better :)